doc/en/user/source/extensions/mapml/template.rst

```diff
@@ -0,0 +1,199 @@
+              <map-geometry>
+                <map-point>
+                  <map-coordinates><#list gattribute.rawValue.coordinates as coord>
+                    <#if coord?index == 0><![CDATA[<span class="desired">]]>${coord.x} ${coord.y}<![CDATA[</span>]]><#else>${coord.x} ${coord.y}</#if></#list></map-coordinates></map-point>
+                </map-geometry>
+             </#if>
+           </#list>
+          </map-feature>
+        </map-body>
+        </mapml->
+
+This would result in a MapML feature output body that would resemble:
+    
+    <mapml- xmlns="http://www.w3.org/1999/xhtml"><map-head><map-title>Bridges</map-title><map-meta charset="UTF-8"/><map-meta content="text/mapml" http-equiv="Content-Type"/><map-meta name="extent" content="top-left-longitude=-180.000000,top-left-latitude=90.000000,bottom-right-longitude=180.000000,bottom-right-latitude=-90.000000"/><map-meta name="cs" content="gcrs"/><map-meta name="projection" content="MapML:urn:ogc:def:crs:EPSG::4326"/><map-style>.bbox {display:none} .Bridges-r1-s1{r:48.0; stroke-opacity:1.0; stroke-dashoffset:0; well-known-name:square; stroke-width:1.0; opacity:1.0; fill:#808080; fill-opacity:1.0; stroke:#000000; stroke-linecap:butt}</map-style></map-head><map-body><map-feature id="Bridges.1107531599613" class="Bridges-r1-s1"><map-geometry><map-point><map-coordinates>&lt;span class="desired">0.0007 0.0002&lt;/span></map-coordinates></map-point></map-geometry><map-properties><table xmlns="http://www.w3.org/1999/xhtml"><thead><tr><th role="columnheader" scope="col">Property name</th><th role="columnheader" scope="col">Property value</th></tr></thead><tbody><tr><th scope="row">FID</th><td itemprop="FID">110</td></tr><tr><th scope="row">NAME</th><td itemprop="NAME">Cam Bridge</td></tr></tbody></table></map-properties></map-feature></map-body></mapml->
+
+Note that in addition to tagging the coordinates with a style class, the template also changes the name of the NAME property to "UPDATED NAME" and the value to "CHANGED Cam Bridge".  
+
+A more complex example mapml-feature.ftl that operates on a multi-polygon layer and only inserts span tags into a specific feature::
+
+    <mapml- xmlns=\"http://www.w3.org/1999/xhtml\">
+        <map-head>
+        </map-head>
+        <map-body>
+        <map-feature>
+        <#if attributes.FID.value == \"117">
+          <#list attributes as attribute>
+            <#if attribute.isGeometry>
+              <map-geometry>
+                <map-multipolygon>
+              <#list 0 ..< attribute.rawValue.getNumGeometries() as index>
+                <#assign polygon = attribute.rawValue.getGeometryN(index)>
+               <map-polygon>
+               <#assign shell = polygon.getExteriorRing()><map-coordinates><#list shell.coordinates as coord><#if coord?index == 0><![CDATA[<span class="desired">]]>${coord.x} ${coord.y}<#elseif coord?index == 4> ${coord.x} ${coord.y}<![CDATA[</span>]]><#else> ${coord.x} ${coord.y}</#if></#list></map-coordinates>
```

Comment:
Should be \<map-span>, or feel free to use \<map-a href="example.org" target="_top">${coord.x} ${coord.y}\</map-a> if you like

Reply:
fixed